WebReflection/udomdiff
# Trying out moveBefore instead of insertBefore

This MR is here to simply provide a testing branch for `moveBefore` proposal, having a way to compare performance or results against current `insertBefore` implementation.

## How to run tests

First of all, once forked, `npm i` to get dependencies, then `npm run build` to create the *dist* version, then:

```sh
npx static-handler .
```

Now reach http://localhost:8080/test/ and check the console
